com.playeveryware.eos/Runtime/Core/Utility/FileSystemUtility.cs

```diff
@@ -526,10 +513,27 @@ public static async Task<string> ReadAllTextAsync(string path)
                 Debug.LogException(e);
                 throw;
             }
+        }
 #endif
+
+        /// <summary>
+        /// Reads all text from the indicated file.
+        /// </summary>
+        /// <param name="path">Filepath to the file to read from.</param>
+        /// <returns>The contents of the file at the indicated path as a string.</returns>
+        public static string ReadAllText(string path)
+        {
+#if UNITY_ANDROID && !UNITY_EDITOR
+            return AndroidFileIOHelper.ReadAllText(path);
+#else
+            return Task.Run(() => ReadAllTextAsync(path)).GetAwaiter().GetResult();
```

Comment:
This should be changed to not use the Async function, but to use the synchronous function instead.

Reply:
This has been done now. Resolving conversation.